tests/test_ext_autosummary.py

```diff
@@ -306,8 +306,12 @@ def test_autosummary_generate_content_for_module_imported_members(app):
                                   'quuz', 'qux']
     assert context['functions'] == ['bar']
     assert context['all_functions'] == ['_quux', 'bar']
-    assert context['classes'] == ['Class', 'Foo']
-    assert context['all_classes'] == ['Class', 'Foo', '_Baz']
+    # changed the meaning of automosummary_imported flag to include only sub-modules
```

Comment:
It would be better to have a distinct test root where you automatically include submodules or when you don't. Additionally, you probably need to document the new template variable ``modules``, and give some examples (and probably add a CHANGES entry). 

It is said "changed the meaning of autosummary_imported flag", but what was changed and why was it changed ? Also why was the 'Class' detected before and not any more after your addition ?